### ISSUE TITLE
Proposal: rename this package to ember-cli-d3-shim

### ISSUE THREAD

**Issue by ivanvanderbyl** [ivanvanderbyl/ember-cli-d3-primitive#22]
Considering this now just wraps `d3@4.1.1`, I think it's time to deprecate `ember-cli-d3-shape` and release `ember-cli-d3-shim@4.1.1`. 

This would require notifying everyone who is using this as a dependency, which thankfully is only [4 projects](http://registry.npmjs.org/-/_view/dependedUpon?group_level=3&startkey=%5B%22ember-cli-d3-shape%22%5D&endkey=%5B%22ember-cli-d3-shape%22%2C%7B%7D%5D&skip=0&limit=1000)

**Comment by mike-north** [comment]
> This would require notifying everyone who is using this as a dependency, which thankfully is only 4 projects

There are probably some private projects that use this :)

Also, there's also a d3-shim (https://github.com/brzpegasus/ember-d3). Do you see a compelling reason to make another library?

**Comment by ivanvanderbyl** [comment]
No, I'd like to either combine this with `ember-d3` or `ember-cli-d3`, however the latter is not just a loader, it includes some helpers and components which should be part of a graphing addon. 

So to help reduce confusion about what this does, I'd like to get rid of the `-shape` in the name :)

**Comment by mike-north** [comment]
I'm sure @mingcodes would have no objection to using a common shim in `ember-cli-d3` to actually import the library and add it to the asset pipeline. His library overlaps a bit with maximum plaid, but as you say this is the "graphing addon" layer -- not the uninteresting "import a third party library" layer.

**Comment by mrosenberg** [comment]
I'm really excited by all the advances made in charting and Ember. I like the idea of renaming this package. We are currently moving away from the ember-charts library to more customized charts and the Ember addon d3 space is confusing as to what each package actually does.

**Comment by ivanvanderbyl** [comment]
@mrosenberg we're working towards making the ecosystem more consistent. It looks like this library will be merged in to `ember-d3` soon: https://github.com/brzpegasus/ember-d3/pull/8
